pkg/config/config.go

```diff
@@ -1601,6 +1602,8 @@ func ResolveSecrets(config Config, origin string) error {
 		config.GetInt("secret_backend_timeout"),
 		config.GetInt("secret_backend_output_max_size"),
 		config.GetBool("secret_backend_command_allow_group_exec_perm"),
+		config.GetString("secret_backend_command_sha256"),
+		config.ConfigFileUsed(),
```

Comment:
Can you explain what `ConfigFileUsed` is for? I haven't seen it before

Reply:
This corresponds to this note in the description:
> Note that when secret_backend_command_sha256 is used the datadog.yaml config has to be restricted as follows:
Must be owned by root and cannot have writable others' permissions on *nix.
Cannot be writable by principals other than Local System and Administrators on Windows.